corehq/blobs/tests/test_migrate.py

```diff
@@ -29,6 +29,7 @@
 class BaseMigrationTest(TestCase):
```

Comment:
Would `TestMigrateBackend` (lower in this file) benefit from these same super calls too? If not, can you explain why or point me to some documentation that explains why?